Automattic/wp-calypso
# NUX: Added the preview steps to the onboarding flow.

## 🚧 Hold off on merging until project dependencies are met

### Changes proposed in this Pull Request

We're adding a [business-only flow](https://github.com/Automattic/wp-calypso/pull/30336/files#diff-28b03eff6ef567c591024d6033cda11dR117) that features the site preview and separate site information steps.

![jan-23-2019 16-15-13](https://user-images.githubusercontent.com/6458278/51584584-2452bb80-1f2a-11e9-8d9f-a744fe2f6e3a.gif)

### Testing instructions
1. Fire up the branch and head to http://calypso.localhost:3000/start
2. Switch to the `'onboarding'`variation for the `improvedOnboarding` AB Test
3. Dive into the flows for each site type at http://calypso.localhost:3000/start/site-type, create a few sites. 

The site preview should only display when you select '**Business**' at the site-type step

The site information steps – title, address and phone – should be separate steps for Business. All other site types should only ask for the site title.
